bcgov/biohubbc-platform
# BHBC-2224 - See Dataset Documents

# Overview
Adds the ability to see all the submitted document / file artifacts associated with a given dataset + the ability to download particular artifacts.

## Links to Jira tickets

 - https://quartech.atlassian.net/browse/BHBC-2224

## Description of relevant changes
 - Added NPM package for MUI Data Grid + require peer dependencies
 - Added an endpoint to download artifact records
 - Added a table to the Dataset page to show dataset artifacts

## Testing Procedure
1. Create a project and survey in SIMS
2. Add one or more attachments to the survey
3. Click **Submit Data**
4. Click **SELECT ALL** and then submit.
5. In BioHub, navigate to the survey you submitted: **Map Search** -> **Species Inventory Project** -> **Find Data**

You should be able to:
 - [ ] View all the documents you submitted
 - [ ] Download a document by clicking the Action button, then **Download Document**
